Sources/Web3Auth/Types.swift

```diff
@@ -200,7 +226,7 @@ public struct W3ALoginParams: Codable {
     }
 
     let loginProvider: String?
-    let dappShare: String?
+    var dappShare: String?
```

Comment:
why is this var?

Reply:
because we insert the dappShare value inside the struct after the user has given us the w3a login params so we need the dappShare to mutable